api/service/version_image_service.go

```diff
@@ -43,6 +44,13 @@ func (s *versionImageService) GetImage(ctx context.Context, model *models.Model,
 	}
 
 	versionImage := imageBuilder.GetVersionImage(ctx, model.Project, model, version)
+
+	jobStatus, err := imageBuilder.GetImageBuildingJobStatus(ctx, model.Project, model, version)
+	if err != nil {
+		log.Errorf("error getting image building job status: %v", err)
```

Comment:
in the event of dependencies conflict, this will not be propagated via the sdk right?